powerauth-webflow-authentication/pom.xml

```diff
@@ -115,6 +115,12 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>javax.xml.bind</groupId>
```

Comment:
🤔 Do we really need the`javax` dependency? It was also migrated to `jakarta`.
```xml
<groupId>jakarta.xml.bind</groupId>
<artifactId>jakarta.xml.bind-api</artifactId>
<version>4.0.0</version>
```

Reply:
I will remove this dependency, it should not be needed anymore.